src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -3697,6 +3697,10 @@ def update_mc_profile_preview(self) -> ManagedCluster:
         mc = self.update_network_plugin_settings(mc)
         # update outbound type
         mc = self.update_outbound_type_in_network_profile(mc)
+        # update loadbalancer profile
+        mc = self.update_load_balancer_profile(mc)
+        # update natgateway profile
+        mc = self.update_nat_gateway_profile(mc)
```

Comment:
No need to call these functions again, they've been called in `self.update_mc_profile_default() `

Reply:
the implementation is not added in azure-cli 2.49

Reply:
Removed. Thanks!